script/install_oneapi.sh

```diff
@@ -31,7 +30,23 @@ then
     sudo -E apt-cache pkgnames intel
     echo "################################"
 
-    travis_retry sudo apt-get install -y intel-oneapi-compiler-dpcpp-cpp-and-cpp-classic intel-oneapi-mkl-devel intel-oneapi-openmp intel-oneapi-tbb-devel
+    # Intel mixes different version numbers in releases. Make sure we install the versions that are compatible.
+    if [ "${ALPAKA_CI_ONEAPI_VERSION}" == "2023.1.0" ]
+    then
+        ALPAKA_CI_TBB_VERSION="2021.9.0"
+    elif [ "${ALPAKA_CI_ONEAPI_VERSION}" == "2023.2.0" ]
+    then
+        ALPAKA_CI_TBB_VERSION="2021.10.0"
+    fi
```

Comment:
Maintaining this table will be hard. Isn't there a way to install the respective TBB for a given oneAPI version?

Reply:
At the moment the compiler pulls in the correct TBB version as dependency. But this could change in the future and since we like to make everything explicit in alpaka...